test/openspiel_test.py

```diff
@@ -0,0 +1,159 @@
+"""Tests the functionality of the OpenspielWrapper on openspiel envs."""
+
+import numpy as np
+import pyspiel
+import pytest
+
+from shimmy import OpenspielWrapper
+
+_PASSING_GAMES = [
+    "2048",
+    "amazons",
+    "backgammon",
+    "bargaining",
+    "blackjack",
+    "breakthrough",
+    "bridge",
+    "catch",
+    "checkers",
+    "chess",
+    "cliff_walking",
+    "clobber",
+    "colored_trails",
+    "connect_four",
+    "coop_to_1p",
+    "cursor_go",
+    "dark_chess",
+    "dark_hex",
+    "dark_hex_ir",
+    "deep_sea",
+    "first_sealed_auction",
+    "gin_rummy",
+    "go",
+    "hanabi",
+    "havannah",
+    "hex",
+    "kriegspiel",
+    "kuhn_poker",
+    "leduc_poker",
+    "lewis_signaling",
+    "liars_dice",
+    "liars_dice_ir",
+    "mancala",
+    "mfg_crowd_modelling",
+    "mfg_crowd_modelling_2d",
+    "mfg_garnet",
+    "negotiation",
+    "nim",
+    "othello",
+    "oware",
+    "pentago",
+    "phantom_go",
+    "phantom_ttt",
+    "phantom_ttt_ir",
+    "pig",
+    "quoridor",
+    "rbc",
+    "skat",
+    "solitaire",
+    "stones_and_gems",
+    "tic_tac_toe",
+    "tiny_bridge_2p",
+    "tiny_bridge_4p",
+    "tiny_hanabi",
+    "trade_comm",
+    "ultimate_tic_tac_toe",
+    "universal_poker",
+    "y",
+]
+
+_FAILING_GAMES = [
+    "battleship",
+    "blotto",
+    "bridge_uncontested_bidding",
+    "coin_game",
+    "coop_box_pushing",
+    "coordinated_mp",
+    "efg_game",
+    "euchre",
+    "goofspiel",
+    "hearts",
+    "laser_tag",
+    "markov_soccer",
+    "matching_pennies_3p",
+    "matrix_cd",
+    "matrix_coordination",
+    "matrix_mp",
+    "matrix_pd",
+    "matrix_rps",
+    "matrix_rpsw",
+    "matrix_sh",
+    "matrix_shapleys_game",
+    "mfg_dynamic_routing",
+    "misere",
+    "morpion_solitaire",
+    "nfg_game",
+    "normal_form_extensive_game",
+    "oh_hell",
+    "oshi_zumo",
+    "pathfinding",
+    "repeated_game",
+    "restricted_nash_response",
+    "sheriff",
+    "start_at",
+    "tarok",
+    "turn_based_simultaneous_game",
+]
+
+
+@pytest.mark.parametrize("game", _PASSING_GAMES)
+def test_all_games(game):
```

Comment:
Add similar test for failing games such that if one of the games starts passing then we can add it to the list of passing games. We should be able to use `pytest.raises()`